woocommerce/wc-admin
# Activity Panel: Inbox: Add Store, Enable Inbox

See #176 

This builds on the read-only REST API work in https://github.com/woocommerce/wc-admin/pull/507 to actually connect the inbox items (read only for now).

To test
Create a few notes if you haven't already using the https://github.com/Automattic/wc-admin-notes-test-bench/
In wp-admin > WooCommerce note the Inbox is back. Click on it.
Observe the notes.
Profit!
